### PR TITLE
Create Animals Table

### PR DESCRIPTION
**Pull request by Felipeg005**
## In order to achieve project requirements, the next actions were taken:

- [x] Use the correct template to generate the repo.
- [x] Create a database named vet_clinic.
- [x] Create a table named animals with the following columns:
- d: integer
- name: string
- date_of_birth: date
- escape_attempts: integer
- neutered: boolean
- weight_kg: decimal

- [x] Insert the required data:
- [x] Write queries for the following:
- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)

Thanks for your Review 😄

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer,
```
Comment (arslanbisharat) [inline schema.sql]: - [x] [Optional] You can add 
- `int NOT NULL IDENTITY(1,1)`
- `PRIMARY KEY (id)`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (arslanbisharat) [inline queries.sql]: - [x] It should select `name` instead of `*`.

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (arslanbisharat) [inline queries.sql]: It should select `name` instead of `*`.

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = 'true';
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (arslanbisharat) [inline queries.sql]: - [x] It should select `name `instead of `*`.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: schema.sql (hunk 1 above)
Comment (Felipeg005) [reply schema.sql]: Great Idea, thanks for this advice 😄

File: queries.sql (hunk 4 above)
Comment (Felipeg005) [reply queries.sql]: Perfect, Thank you a lot everything it's changed to name 👍

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Felipeg005, 

Great work on making the changes requested by the previous code reviewer.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**